src/ray/object_manager/plasma/object_spill_manager.h

```diff
@@ -0,0 +1,92 @@
+// Copyright 2017 The Ray Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#pragma once
+
+#include <string>
+#include <vector>
+
+#include "absl/container/flat_hash_map.h"
+#include "absl/strings/str_format.h"
+#include "absl/types/optional.h"
+#include "ray/common/asio/instrumented_io_context.h"
+#include "ray/common/id.h"
+#include "ray/object_manager/plasma/common.h"
+#include "ray/object_manager/plasma/spill_worker_pool.h"
+#include "src/ray/protobuf/common.pb.h"
+
+namespace plasma {
+
+class ObjectSpillManager {
+ public:
+  ObjectSpillManager(instrumented_io_context &callback_executor,
+                     std::unique_ptr<ISpillWorkerPool> spill_worker_pool,
+                     int64_t max_active_worker);
+
+  /// Spill a list of objects in an asynchronous task.
```

Comment:
`asynchronous task.` sounds a bit confusing. What about calling the method as `SubmitSpillingRequest` and comment as `Submit a request to spill a list of objects`?